### PR TITLE
Refactor variable names for Azure Cloud Provide

### PR DESCRIPTION
**Pull request by ilausuch**
- Related ticket: http://progress.opensuse.org/issues/98379
- Verification run:  https://openqa.suse.de/tests/7843258

### CODE REVIEW

**Review by pdostal — APPROVED** [review]
This is awesome!

File: variables.md [hunk 1]
```diff
@@ -173,12 +173,12 @@ PUBLIC_CLOUD_REGION | string | "" | The region to use. (default-azure: westeurop
 PUBLIC_CLOUD_RESOURCE_NAME | string | "openqa-vm" | The name we use when creating our VM.
 PUBLIC_CLOUD_SKIP_MU | boolean | false | Debug variable used to run test without maintenance updates repository being applied.
 PUBLIC_CLOUD_SERVICE_ACCOUNT | string | "" | GCE only, used to specify the service account.
-PUBLIC_CLOUD_TENANT_ID | string | "" | This is B<only for azure> and used to create the service account file.
+PUBLIC_CLOUD_AZURE_TENANT_ID | string | "" | This is B<only for azure> and used to create the service account file.
```
Comment (jlausuch) [inline variables.md:176]:
```suggestion
PUBLIC_CLOUD_AZURE_TENANT_ID | string | "" | Used to create the service account file.
```

File: variables.md [hunk 2]
```diff
@@ -173,12 +173,12 @@ PUBLIC_CLOUD_REGION | string | "" | The region to use. (default-azure: westeurop
...
 PUBLIC_CLOUD_TOOLS_REPO | string | false | The URL to the cloud:tools repo (optional). (e.g. http://download.opensuse.org/repositories/Cloud:/Tools/openSUSE_Tumbleweed/Cloud:Tools.repo).
 PUBLIC_CLOUD_TTL_OFFSET | integer | 300 | This number + MAX_JOB_TIME equals the TTL of created VM.
 PUBLIC_CLOUD_SLES4SAP | boolean | false | If set, sles4sap test module is added to the job.
 PUBLIC_CLOUD_STORAGE | string | "openqa-storage" | Name of storage where we load the system image from.
-PUBLIC_CLOUD_SUBSCRIPTION_ID | string | "" | This is B<only for azure> and used to create the service account file.
+PUBLIC_CLOUD_AZURE_SUBSCRIPTION_ID | string | "" | This is B<only for azure> and used to create the service account file.
```
Comment (jlausuch) [inline variables.md:181]:
```suggestion
PUBLIC_CLOUD_AZURE_SUBSCRIPTION_ID | string | "" | Used to create the service account file.
```

Comment (ilausuch) [reply variables.md]: Done

**Comment by ilausuch** [comment]
This can be merged without changes in QAC OpenQA yaml

**Review by jlausuch — APPROVED** [review]

**Pull request merged**